patches/chrome-browser-resources-md_extensions-item_list.html.patch

```diff
@@ -10,7 +10,7 @@ index 90244de120cf8fda71002f535810619b1e8aeb3e..84450aca34abf7b55e1289b8716b81d5
 +            hidden$="[[shouldShowEmptyItemsMessage_(
```

Comment:
can we create a template for this so it's just a single line patch? See chrome/browser/resources/md_bookmarks/item.html / chrome/browser/resources/md_bookmarks/list.html

Reply:
I don't think this was added here

Reply:
I will make a separate issue to address this.